qiskit_nature/drivers/second_quantization/gaussiand/gaussian_forces_driver.py

```diff
@@ -86,6 +86,7 @@ def from_molecule(
         molecule: Molecule,
         basis: str = "sto-3g",
         driver_kwargs: Optional[dict[str, Any]] = None,
+        xcf: str = "B3LYP",
```

Comment:
This is a legacy file which should no longer receive any changes. Please revert the changes done to this file